gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/dataset/ConvertibleHiveDataset.java

```diff
@@ -211,16 +215,29 @@ private ConversionConfig(Config config, Table table, String destinationFormat) {
       this.destinationDataPath = resolveTemplate(config.getString(DESTINATION_DATA_PATH_KEY), table);
 
       // Optional
-      this.destinationTableProperties = ConfigUtils
-          .configToProperties(ConfigUtils.getConfig(config, DESTINATION_TABLE_PROPERTIES_KEY, ConfigFactory.empty()));
+      this.destinationTableProperties =
+          convertKeyValueListToProperties(ConfigUtils.getStringList(config, DESTINATION_TABLE_PROPERTIES_LIST_KEY));
       this.clusterBy = ConfigUtils.getStringList(config, CLUSTER_BY_KEY);
       this.numBuckets = Optional.fromNullable(ConfigUtils.getInt(config, NUM_BUCKETS_KEY, null));
-      this.hiveRuntimeProperties = ConfigUtils
-          .configToProperties(ConfigUtils.getConfig(config, HIVE_RUNTIME_PROPERTIES_KEY_PREFIX, ConfigFactory.empty()));
+
+      this.hiveRuntimeProperties =
```

Comment:
This wont work for values that are comma separated list (for eg. search for 'comma' on this page): https://cwiki.apache.org/confluence/display/Hive/Configuration+Properties